man/systemd-vmspawn.xml

```diff
@@ -34,6 +34,8 @@
     <para><command>systemd-vmspawn</command> may be used to start a virtual machine from an OS image. In many ways it is similar to <citerefentry
     project='man-pages'><refentrytitle>systemd-nspawn</refentrytitle><manvolnum>1</manvolnum></citerefentry>, but it
     launches a full virtual machine instead of using namespaces.</para>
+
+  <para>Note: on Ubuntu/Debian derivatives systemd-vmspawn requires the user to be in the <literal>kvm</literal> group to use the VSock options.</para>
```

Comment:
Hmm, I'm not sure if we should document distro-specific things here. Also indentation is off

Reply:
This problem is very common and will affect a lot of users, so it's worth calling it out

Reply:
I have fixed the indentation in 6b30cad5e9.